### PR TITLE
Projectkk2glider/fatfs 0.12

### PR DESCRIPTION
**Pull request by projectkk2glider**

### CODE REVIEW

File: radio/src/gui/128x64/radio_sdmanager.cpp [hunk 1]
```diff
@@ -220,16 +217,20 @@ void menuRadioSdManager(event_t _event)
 
     FRESULT res = f_opendir(&dir, "."); // Open the directory
     if (res == FR_OK) {
+      bool firstTime = true;
```
Comment (bsongis) [inline radio/src/gui/128x64/radio_sdmanager.cpp]: But then we will have a ".." even in the root directory?

File: radio/src/thirdparty/FatFs/00readme.txt [hunk 2]
```diff
@@ -1,163 +1,21 @@
...
 
-  ffconf.h   Configuration file for FatFs module.
-  ff.h       Common include file for FatFs and application module.
-  ff.c       FatFs module.
-  diskio.h   Common include file for FatFs and disk I/O module.
-  diskio.c   An example of glue function to attach existing disk I/O module to FatFs.
-  integer.h  Integer type definitions for FatFs.
-  option     Optional external functions.
+  00readme.txt This file.
+  history.txt  Revision history.
```
Comment (bsongis) [inline radio/src/thirdparty/FatFs/00readme.txt]: Did you forget to add this file to your commit?

**Comment by projectkk2glider** [comment]
**Warning** this last commit was tested only in `simu`, I don't have a radio available rigth now. I am counting on `f_getcwd()` to return `"/"` for the root dir on the actual radio. Test is simple, if SD card manager shows `[..]` for the first dir, then this does not work properly.

**Comment by kilrah** [comment]
Works OK on the radio

**Comment by bsongis** [comment]
Thanks!

**Pull request merged**